docs/_layouts/global.html

```diff
@@ -20,12 +20,15 @@
         <link rel="stylesheet" href="css/bootstrap.min.css">
         <style>
             body {
+	        {% if page.url contains "/ml" or page.url contains "/sql" or page.url contains "migration-guide.html" %}
```

Comment:
I built the docs to take a look, and wondered, why special case these? the /ml and /sql pages look like they have too little padding at the top.

Reply:
@clarkead I'd even merge this as-is but wondering if the above change is necessary?

Reply:
Thank you for reviewing this.  I believe it was trying to address an issue with the left nav on those pages, but I don't remember the specifics.  I do see the spacing difference though and I agree we should try to fix it.  There should be a way to do this without making an exception as this does.  Thank you for calling it out.  I appreciate the feedback.

In reviewing this I think I might have noticed a display bug on the rdd-programming-guide.html and structured-streaming-programming-guide.html pages when viewed in a mobile device's browser.

Let me take a look at both issues and fix them.  I'll try to submit another update before the end of this week.

Reply:
It looked OK to me otherwise. Maybe it is just a matter of removing the special-casing here.

Reply:
@clarkead I think it's sufficient (for now) to just remove the special casing code here. It seemed like it was causing the differences, itself, unless I really misunderstand its other effects - does something else go wrong if you don't special case? it seems to raise the title too far, and that's all I'd expect, so removing it would fix it.

Reply:
@srowen Sorry about the delay getting an updated PR.  I've pushed two small tweaks to address the items in this thread.  The first is to try and fix the formatting on those pages with left navigation that had special logic.  I've removed the logic and fixed the width on the expand/collapse trigger.

The second change is with regards to the issue I saw on the rdd-programming-guide.html and streaming-programming-guide.html pages.  It ended up being an issue where older versions of Bootstrap where setting the max-width on images to 100%, but Bootstrap4 removed that.  As a result larger image files were blowing out the right side.  I've added a css style to re-add this in main.css.